scripts/variantstore/wdl/extract/import_gvs.py

```diff
@@ -312,18 +312,30 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
     info(f'import_gvs: using target_records (records per partition) of {target_records} for VDS merge')
 
     interval_tmp = os.path.join(tmp_dir, 'interval_checkpoint.ht')
+    if hl.hadoop_exists(interval_tmp):
+        info(f'import_gvs: interval checkpoint table "{interval_tmp}" already exists, deleting')
+        hl.current_backend().fs.rmtree(interval_tmp)
+
     target_final_intervals, _ = calculate_new_intervals(first_ref_mt, target_records, interval_tmp)
 
     with hl._with_flags(no_whole_stage_codegen='1'):
 
         merge_tmp = os.path.join(tmp_dir, 'merge_tmp.vds')
-        info(f'import_gvs: calling Hail VDS combiner for merging {len(vds_paths)} intermediates')
-        combiner = hl.vds.new_combiner(output_path=merge_tmp,
-                                       vds_paths=vds_paths,
-                                       target_records=target_records,
-                                       temp_path=tmp_dir,
-                                       use_genome_default_intervals=True)
-        combiner.run()
+        from hail.vds import VariantDataset
+        ref_success_path = os.path.join(VariantDataset._reference_path(merge_tmp), '_SUCCESS')
+        var_success_path = os.path.join(VariantDataset._variants_path(merge_tmp), '_SUCCESS')
+        if hl.hadoop_exists(ref_success_path) and hl.hadoop_exists(var_success_path):
+            info(f'import_gvs: Hail VDS combiner is done. Skipping it')
+        else:
+            info(f'import_gvs: calling Hail VDS combiner for merging {len(vds_paths)} intermediates')
+            combiner = hl.vds.new_combiner(output_path=merge_tmp,
+                                           vds_paths=vds_paths,
+                                           target_records=target_records,
+                                           branch_factor=52,  # Echo has 104 intermediate VDSes so 2 equal sized groups
```

Comment:
here's the branch factor that we discussed. 52 is great for now, but will almost certainly need to be changed for Foxtrot

Reply:
Leaving this at 52 for now, adding breadcrumbs in the code